snap/hooks/install

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash -e
+
+# install all the config files from $SNAP/config/SERVICE/res/configuration.toml 
+# into $SNAP_DATA/config
+mkdir -p ${SNAP_DATA}/config
+for service in security-gateway-api core-command core-config-seed-go core-data core-metadata export-client export-distro support-logging; do
+    if [ ! -f "${SNAP_DATA}/config/${service}/res/configuration.toml" ]; then
+        mkdir -p "${SNAP_DATA}/config/${service}/res"
+        cp ${SNAP}/config/${service}/res/configuration.toml "${SNAP_DATA}/config/${service}/res/configuration.toml"
+    fi
+done
+
+# special handling for additional core-config-seed-go config toml files
```

Comment:
Why copy config-seed configuration.toml files in $SNAP_DATA?  Do we expect that this is the way admins will make consul based configuration changes?

Reply:
I just looked at every single config file that is installed into the snap and copied all of them into $SNAP_DATA. I don't know if admins will modify these or not, but if we don't put them in $SNAP_DATA they can't be modified at all...